src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/build/AdditionalFiles/vs-threading.MembersRequiringMainThread.txt

```diff
@@ -25,7 +38,17 @@
 ![Microsoft.VisualStudio.Shell.Interop.IVsAppContainerProjectDeployOperation]
 ![Microsoft.VisualStudio.Shell.Interop.IVsAppContainerProjectDeployResult]
 ![Microsoft.VisualStudio.Shell.Interop.SVsAppContainerProjectDeploy]
+![Microsoft.VisualStudio.Shell.Interop.IVsAppQueryLoad]
+![Microsoft.VisualStudio.Shell.Interop.IVsAppQueryLoad2]
+![Microsoft.VisualStudio.Shell.Interop.SVsAppQueryLoad]
```

Comment:
These are private interfaces used by the package manager, its true they are free-threaded by the necessity of having a free-threaded package manager, but really NO ONE other than the package manager should use them, and the package manager, at least currently is written in native code. I'd be worried people take this list as a 'supported public services' list in some way, which these are DEFINETLY not public services.

Reply:
Fixed.

Reply:
Coming back to this, this we have other internal usage here and I see usage of these APIs in code external to the package manager, such as in MEF. I'm going to readd these so that we can be clean.